modules/ublog/src/main/Env.scala

```diff
@@ -43,15 +43,14 @@ final class Env(
     cacheApi.unit[List[UblogPost.PreviewPost]]:
       _.refreshAfterWrite(10 seconds).buildAsyncFuture: _ =>
         import ornicar.scalalib.ThreadLocalRandom
-        val lookInto = 5
-        val keep     = 2
+        val lookInto = 7
         api
           .latestPosts(lookInto)
           .map:
             _.groupBy(_.blog)
               .flatMap(_._2.headOption)
+              .toList
           .map(ThreadLocalRandom.shuffle)
-          .map(_.take(keep).toList)
```

Comment:
Why put 7 posts in the cache when only use 2 or 3?

Reply:
Give clients of the cache a bit more options when formatting. I'm not sure it matters where we throw the tail away though.